backend/scripts/jacoco_coverage.py

```diff
@@ -42,13 +65,42 @@ def fetch_coverage_info(parser: callable, coverage_type: str) -> list:
     return [coverage_type, str(missed), str(total), str(coverage)]
 
 
+def is_coverage_greater_than_min(coverage_type: str, expected_coverage: float) -> (bool, float):
+    """
+    Checks if actual coverage is greater than expected coverage.
+    :returns: coverage_greater_than_min and actual_coverage
+    """
+    coverage_type = coverage_type.lower()
+    actual_coverage = 0
+    if coverage_type == "complexity":
+        actual_coverage = float(COVERAGE_MAP["complexity"][3])
+    elif coverage_type == "line":
+        actual_coverage = float(COVERAGE_MAP["line"][3])
+    elif coverage_type == "method":
+        actual_coverage = float(COVERAGE_MAP["method"][3])
+    elif coverage_type == "class":
+        actual_coverage = float(COVERAGE_MAP["class"][3])
+
+    return actual_coverage >= expected_coverage, actual_coverage
+
+
+def map_coverage_info(coverage_info_array: list) -> None:
```

Comment:
Doesn't feel quite right but I couldn't come up with a better way of doing this.

Reply:
I am at least removing the use of global variable for now.